README.md

```diff
@@ -16,7 +26,23 @@ An website for watching videos together.
 - Text chat
 - Video chat
 
-## Configuration
+## Advanced Setup
+
+### YouTube API
+
+This project is using the YouTube API which means you will need to setup an API key. You can get one from Google [here](https://console.developers.google.com/?hl=de).
```

Comment:
Link is to german version, should be https://console.developers.google.com/ without the ?hl=de

Reply:
Thanks for the catch. :)